src/pages/studyView/StudyViewPage.tsx

```diff
@@ -83,7 +84,7 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                     return;
                 }
 
-                this.store.updateCurrentTab(props.routing.location.query.tab);
+                this.store.updateCurrentTab(getStudyViewTabId(getBrowserWindow().globalStores.routing.location.pathname));
```

Comment:
Maybe the old format should to be updated to the new structure. /study?id=***&tab=summary should be redirected to /study/summary?id=***
@alisman what's your experience in terms of handling old URL? Do we generally ignore it or redirect the tab?